airbyte-integrations/connectors/destination-vectara/destination_vectara/writer.py

```diff
@@ -105,7 +105,7 @@ def _get_document_metadata(self, record: AirbyteRecordMessage) -> Dict[str, Any]
     def _get_document_title(self, record: AirbyteRecordMessage) -> str:
         title = "Untitled"
         if self.title_field:
-            title = dpath.util.get(record.data, self.title_field)
+            title = dpath.util.values(record.data, self.title_field, separator=".")[0]
```

Comment:
Is it possible that the call to `values()` would return zero results, for instance if `title_field` is not found?

If that's feasible (and I'm not sure), then what do you thin of a quick check first? 

Something like...

```suggestion
            found_title = dpath.util.values(record.data, self.title_field, separator=".")
            if found_values:
                title = found_title[0]
```